playbook/app/pb_kits/playbook/pb_star_rating/_star_rating.tsx

```diff
@@ -1,22 +1,20 @@
-/* @flow */
+import React from "react"
+import classnames from "classnames"
 
-import React from 'react'
-import classnames from 'classnames'
+import { buildAriaProps, buildDataProps } from "../utilities/props"
 
-import { buildAriaProps, buildDataProps } from '../utilities/props'
-
-import Icon from '../pb_icon/_icon'
+import Icon from "../pb_icon/_icon"
 
 type StarRatingProps = {
-  aria?: object,
-  className?: string,
-  data?: object,
-  fixedWidth?: boolean,
-  hideRating: boolean,
-  icon?: string,
-  id?: string,
-  rating: number,
-}
+  aria?: {[key: string]: string};
```

Comment:
These should be commas `,` not semicolons. (See _avatar.tsx or _body.tsx)

Reply:
good call I think my code formatter went a little agro here